include/crocoddyl/core/diff-action-base.hpp

```diff
@@ -118,6 +118,9 @@ class DifferentialActionModelAbstractTpl {
    */
   void calcDiff(const boost::shared_ptr<DifferentialActionDataAbstract>& data, const Eigen::Ref<const VectorXs>& x);
 
+  virtual void quasiStatic(const boost::shared_ptr<DifferentialActionDataAbstract>& data, Eigen::Ref<VectorXs> u,
```

Comment:
Why are you declaring it as pure virtual when in the base is declare as virtual?
As before, if you don't plan to implement it, then remove this function declaration.